src/overcooked_ai_py/mdp/overcooked_env.py

```diff
@@ -4,6 +4,7 @@
 from overcooked_ai_py.utils import mean_and_std_err, append_dictionaries
 from overcooked_ai_py.mdp.actions import Action
 from overcooked_ai_py.mdp.overcooked_mdp import OvercookedGridworld, EVENT_TYPES
+from overcooked_ai_py.mdp.overcooked_trajectory import TIMESTEP_TRAJ_KEYS, EPISODE_TRAJ_KEYS, DEFAULT_TRAJ_KEYS
```

Comment:
Thanks for this refactoring here!

Reply:
Still think we should aim for a full-fledged `Trajectory` class at some point, but this should serve as a nice middle ground for now!